README.md

```diff
@@ -69,18 +70,18 @@ var body: some View {
 }
 ```
 
-And just like that, you've got a workflow! You can now add more items to it or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
+And just like that, you've got a workflow! You can now add more items to it or reorder the items that are there. To understand more of how this works, [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Check out our example apps
-We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
+# Try Our Example Apps
+We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They've already been tested, so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
 
-# [See this to learn more](https://wwt.github.io/SwiftCurrent/Creating%20Workflows.html)
+# [See These to Learn More](https://wwt.github.io/SwiftCurrent/Creating%20Workflows.html)
```

Comment:
I think the intent of this wasn't quite hitting on what we were trying to drive the reader to do.

We were hoping that saying "See this to learn more" would drive the reader to click the link to end up on our website with useful documentation where they would continue to read and learn.  The bullets below are additional calls to do the same thing but leading to specific areas of interest.  I.E., You want to install, so go here, but at the title level, you just want to engage more.

By changing this to "See These to Learn More", it feels like we are directing users away from the clickable link and instead to read more below.  🤷  not sure if that's a problem yet...

Thoughts?

Reply:
We can by doing:
```suggestion
# See [This](https://wwt.github.io/SwiftCurrent/Creating%20Workflows.html) to Learn More
```

Reply:
BTW, to make a suggestion like I did above, you can press cmd-g OR click the +- file next to the big H in the editing bar above your comment box.

Reply:
I really want to drive people to that link. I'm a little uncertain about just making "this" a link. We need something that really stands out.

Reply:
@Tyler-Keith-Thompson Perhaps restructuring the link?  You want to drive people to click the link because you want them on the website and off of GitHub, right?  So the goal for this line is:

1. Direct people to the website
2. Entice them to click the link so they end up on the website
3. Act as a section header for even more resources they can find on the website

Does that sound right?  If it is, do we need all 3 of those things to be done in this one section header or if we could make something sufficiently enticing, this could just be a header and something else fulfills goals 1 and 2?

Reply:
Your summation of goals is well stated. I don't have any particular heartburn over how we achieve those goals

Reply:
How about "Click Here for Learn More"

Reply:
Change made manually - resolving conversation